cli/mod.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
+
+export * from "./parse_args.ts";
```

Comment:
How about a short description for this sub-module? Maybe something like "Tools for interacting with the Deno CLI."

Reply:
Happy to approve once this is addressed, @lino-levan.